optional_requirements.txt

```diff
@@ -4,3 +4,4 @@ SimpleITK; python_version != '3.4' and python_version != '3.5'
 astropy
 tifffile
 imageio
+dask[array]
```

Comment:
Why not just to copy-paste it? I don't know the history of `>=0.5` requirement, but I believe it is there for a reason. :)